docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -0,0 +1,97 @@
+Dockstore CLI FAQ
+=================
+
+How does launching with Dockstore CLI compare with cwltool?
```

Comment:
This is really good to capture, I've struggled to explain it. Two thoughts:

1. Make clear that the CLI is invoking cwltool under the hood, that it's not an alternative, but a essentially a wrapper.
2. Although it needs to be synced with the 1.12 release/support, you could also mention the CLI makes it easy to transition running on a WES server.

Reply:
Ha, now I see this was already there, you just moved it. :) I just start my review process from the first file, working my way down... I still think my suggestions apply though.